cMake/FreeCAD_Helpers/SetupSalomeSMESH.cmake

```diff
@@ -130,7 +130,7 @@ macro(SetupSalomeSMESH)
         endif()
 
         set(SMESH_FOUND TRUE)
-        configure_file(SMESH_Version.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/SMESH_Version.h)
+        configure_file(${CMAKE_SOURCE_DIR}/cMake/header/SMESH_Version.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/SMESH_Version.h)
```

Comment:
A typo here still fails the CI: `s|cMake/header|cMake/headers|`

Reply:
@mosfet80 :point_up: